basic_autonomy/src/basic_autonomy.cpp

```diff
@@ -437,6 +443,9 @@ namespace basic_autonomy
             std::vector<double> downtracks_raw_line1 = carma_wm::geometry::compute_arc_lengths(line_1);
             int total_step_along_curve1 = static_cast<int>(downtracks_raw_line1.back() / 2.0);
             double step_threshold_line1 = (double)total_step_along_curve1 / (double)total_point_size;
+            //TODO: are we missing some computation here?  step_threshold_line1 and step_threshold_line2 are not used anywhere
```

Comment:
This calculation looks redundant, because on Line 452 a common scale is being used for both the curves, which is needed to get the same number of points.
I'm a little hesitant to suggest removing it without looking into it further, but doesn't seem like there's a missing step

Reply:
Okay, commented out these lines for now, but leaving the TODO in place so someone will look at it eventually.